testing/mock/mock.go

```diff
@@ -248,6 +286,10 @@ func (am AppModule) NegotiateAppVersion(
 	counterparty channeltypes.Counterparty,
 	proposedVersion string,
```

Comment:
haha nice call @damiannolan on not using `_` for unused fields, since I ended up using them now